libs/membership/membership-impl/src/main/kotlin/net/corda/membership/lib/impl/grouppolicy/v1/MemberGroupPolicyImpl.kt

```diff
@@ -105,6 +107,16 @@ class MemberGroupPolicyImpl(rootNode: JsonNode) : MemberGroupPolicy {
             }
             output.toList()
         }
+
+        override val staticNetworkGroupParameters: Map<String, String>? =
+            staticNetwork?.getOptionalStringMap(GROUP_PARAMETERS)?.apply {
+                val verifierResult = GroupParametersUpdateVerifier().verify(this)
+                if (verifierResult is GroupParametersUpdateVerifier.Result.Failure) {
+                    with(verifierResult.reason) {
+                        throw BadGroupPolicyException(this)
+                    }
```

Comment:
Thjis seems a bit tidier:
```suggestion
                throw BadGroupPolicyException(verifierResult.reason)
```